### PR TITLE
Newb1-please check the code

### PR DESCRIPTION
**Pull request by sai813kiran**
I have made the changes please check the code

### CODE REVIEW

File: src/git/testgit.java [hunk 1]
```diff
@@ -8,6 +8,7 @@ public static void main() {
 		
 System.out.println("hello");
 System.out.println("king");
+System.out.println("this is test local");
```
Comment (SaiKiran9802) [inline src/git/testgit.java:11]: please add the hello

Comment (sai813kiran) [reply src/git/testgit.java:11]: okay

Comment (sai813kiran) [reply src/git/testgit.java:11]: I have added please check now

**Comment by SaiKiran9802** [comment]
closed

**Pull request closed without merging**